src/integration/java/com/smartcar/sdk/VehicleTest.java

```diff
@@ -228,4 +228,15 @@ public void testBatchResponseHeaders() throws SmartcarException, BatchResponseMi
     public void testDisconnect() throws SmartcarException {
         this.vehicle.disconnect();
     }
+
+    /**
+     * Tests setting the api version to 2.0 and getting the api url that is used for subsequent requests
+     */
+    @Test
+    public void testSetApiVersion() throws SmartcarException {
+        this.vehicle.setApiVersion("2.0");
+        String url = Vehicle.getApiUrl();
```

Comment:
should we add a similar test to the auth client?

Reply:
yep!